### ISSUE TITLE
Published package doesn't include build files

### ISSUE THREAD

**Issue by xeho91** [jo-sm/stylelint_d#100]
Hi there!

Thank you very much for this daemon; I've been using it for a while.
Recently I decided to update it, and I encountered a problem.

After installing your package with:
```sh
pnpm i -g stylelint_d
```
I get the following error:
```log
Packages: +4
++++
Progress: resolved 1614, reused 1614, downloaded 0, added 0, done
node_modules/.pnpm/node-pty@0.10.1/node_modules/node-pty: Running install script, done in 1.7s
node_modules/.pnpm/node-pty@0.10.1/node_modules/node-pty: Running postinstall script, done in 53ms
 ERROR  ENOENT: no such file or directory, chmod '/usr/pnpm-global/5/node_modules/stylelint_d/dist/bin/stylelint_d.js'
```

I have tested with using `npm`, same problem:
```log
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /usr/lib/node_modules/stylelint_d/dist/bin/stylelint_d.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/lib/node_modules/stylelint_d/dist/bin/stylelint_d.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-07-09T12_54_08_467Z-debug.log
```

I inspected the [./package.json](https://github.com/jo-sm/stylelint_d/blob/master/package.json), and since you have moved to TypeScript, you didn't specify which files should be published as a package. The content of the installed package is:

**`pwd`**:  `/usr/pnpm-global/5/node_modules/stylelint_d`

**`tree -a --dirsfirst`**:

```log
.
├── node_modules
│   └── .bin
│       └── stylelint
├── LICENSE.txt
├── package.json
├── .prettierignore
└── README.md

2 directories, 5 files
```

Without having to reorganize your project's structure. The quick solution would be adding the code below to the [./package.json](https://github.com/jo-sm/stylelint_d/blob/master/package.json).
```json
  ...
  "files": [
    "dist/**/*"
  ]
  ...
```

**Comment by jo-sm** [comment]
Hey @xeho91 thanks for pointing this out, and sorry for the trouble! I can't believe this slipped past me unnoticed for so long, considering I published 2.2.0 almost two weeks ago 🤦 

The problem was that I had published the package manually and incorrectly. I set up a CI step to handle the build+publish step a while ago but this most recent publish I had done it locally and I get the feeling I forgot to actually run the proper `npm run build` command... In any case, I've published `2.2.1` which fixes the issue, and verified that it works for me running `npm i -g stylelint_d` - can you try it?

I'm going to make a new issue related to this later today so that I can prevent this specific issue from happening in the future.

**Comment by xeho91** [comment]
Yes, it works now. Thank you, and what a quick response. Kudos!
